visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -173,7 +189,8 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
     constructor(props: IModelComparisonProps) {
         super(props);
         this.state = {
-            disparityInOutcomes: true
+            disparityInOutcomes: true,
```

Comment:
this should not be part of the state anymore, instead should set the default disparityKey?